backup/moodle2/backup_moodleoverflow_stepslib.php

```diff
@@ -45,25 +45,26 @@ protected function define_structure() {
 
         // Define the root element describing the moodleoverflow instance.
         $moodleoverflow = new backup_nested_element('moodleoverflow', ['id'], [
-            'name', 'intro', 'introformat', 'maxbytes', 'maxattachments',
-            'forcesubscribe', 'trackingtype', 'timecreated', 'timemodified',
-            'ratingpreference', 'coursewidereputation', 'allownegativereputation']);
+            'name', 'intro', 'introformat', 'maxbytes', 'maxattachments', 'timecreated', 'timemodified',
+            'forcesubscribe', 'trackingtype', 'ratingpreference', 'coursewidereputation', 'allowrating',
+            'allowreputation', 'allownegativereputation', 'grademaxgrade', 'gradescalefactor', 'gradecat',
+            'anonymous', 'needsreview', 'allowmultiplemarks', ]);
 
         // Define each element separated.
         $discussions = new backup_nested_element('discussions');
         $discussion = new backup_nested_element('discussion', ['id'], [
-            'name', 'firstpost', 'userid', 'timemodified', 'usermodified', 'timestart']);
+            'name', 'firstpost', 'userid', 'timemodified', 'usermodified', 'timestart', ]);
 
         $posts = new backup_nested_element('posts');
 
         $post = new backup_nested_element('post', ['id'], [
             'parent', 'userid', 'created', 'modified',
-            'mailed', 'message', 'messageformat', 'attachment']);
+            'mailed', 'message', 'messageformat', 'attachment', ]);
```

Comment:
misses reviewed and time reviewed in case we want to also backup that

Reply:
?

Reply:
is added